src/wp-includes/class-wp-script-modules.php

```diff
@@ -151,6 +151,20 @@ public function dequeue( string $id ) {
 		unset( $this->enqueued_before_registered[ $id ] );
 	}
 
+	/**
+	 * Deregister the script module so it will no longer be registered.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param string $id The identifier of the script module.
+	 */
+	public function deregister( string $id ) {
+		if ( isset( $this->registered[ $id ] ) ) {
```

Comment:
`unset` is a language construct so it isn’t necessary to wrap it with `isset` check.

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/6061/commits/11d2ea7ecbec9b63f1d20fbda5007397b3153b95